### PR TITLE
Rails 5.1 compatibility

### PR DESCRIPTION
**Pull request by workgena**

### CODE REVIEW

File: lib/active_admin_scoped_collection_actions/dsl.rb [hunk 1]
```diff
@@ -53,7 +53,7 @@ def add_scoped_collection_action_default_destroy(_, &block)
       batch_action :scoped_collection_destroy, if: proc { false } do |_|
         unless authorized?(:batch_destroy, resource_class)
           flash[:error] = I18n.t('active_admin_scoped_collection_actions.actions.no_permissions_msg')
-          render nothing: true, status: :no_content and next
+          head :no_content and next
```
Comment (Fivell) [inline lib/active_admin_scoped_collection_actions/dsl.rb]: @workgena , looks like proper way is 
``head :ok```

or 

```
render body: nil
```

Comment (Fivell) [inline lib/active_admin_scoped_collection_actions/dsl.rb]: https://github.com/rails/rails/pull/20336/files

**Comment by Fivell** [comment]
@workgena , also I think minamal ruby version should be 2.2.7 in travis (like in AA 1.0)

File: lib/active_admin_scoped_collection_actions/dsl.rb (hunk 1 above)
Comment (workgena) [inline lib/active_admin_scoped_collection_actions/dsl.rb]: yes, it will be better

**Comment by workgena** [comment]
changed versions to: Ruby 2.2.7 and 2.3.4

**Pull request merged**